### ISSUE TITLE
Magnet Card not working properly

### ISSUE THREAD

**Issue by PhilGooodInc** [p455w0rd/WirelessCraftingTerminal#159]
The magnet card works for a while, then all of a sudden, just stops inserting into the ME System (while using the infinty card) but still magnets the items into my inventory. I have to close the instance and restart it completely so it can work again. Have tried turning magnet off, removing the card then reinserting, nothing.  Even a reload isnt enough. I have no idea what triggers it. Yesterday it worked for a couple of hours, today, it barely works for 5 minutes.

I am using Age of Engineering pack 1.04 and updated to 1.32 version of WCT. The previous version was even more buggy.

**Comment by p455w0rd** [comment]
Previous version being buggy shouldn't be asurprise..this is why we do updates..as far as the magnet card..use twitch client to create a new instance with the bare minimum for WCT (AE2, WCT, PWLib) and let me know how it functions...I've had this report many times but cannot reproduce using this setup meaning it's mod interaction at some point..could be on my end or another mod's end...tbh, I'll probably just remove the magnet...most people just use DE, Botania, or EIO magnet anyway

**Comment by Miner94** [comment]
this only happened to me when i unloaded the chunks that AE2 is in, Other than that it does not interact with botania's magnet at all

**Comment by p455w0rd** [comment]
So the magnet is actually sort of intelligent in certain ways. If you throw an item on the ground, the magnet detects this and will not pick the item up for a short amount of time. It does this to enable throwing things into lava or tossing items to another player. Beyond this, most times this has been reported, it is as @Miner94 said..network not being loaded/having power/etc. In 1.10.2 WCT, I have added new functionality that turns the AE2 controller into a chunkloader to alleviate some of these unwarranted reports, but the chunkloading functionality is configurable and may be turned off. So even with the AE2 controller being chunkloaded, if part of your network becomes unloaded (is in a chunk separate from the controller), it would also cause this to happen. I have just done testing with the magnet and it appears to be functioning as intended. As you aren't replying, I'm going to close this issue. If you run all of these tests and you still feel this is a bug, you're welcome to reopen the issue. On another note, there was some buggyness when the WCT was being used as a bauble. Azanor has implemented autosyncing, but it only goes from server->client. For certain situations, the syncing needs to go the other direction, and with the latest update, I have added this functionality, so this may very well fix your issue (it has resolved a few). At any rate, I do appreciate your taking the time to make the report and help me in trying and keep this mod bug free :)
